gradle/versions.gradle

```diff
@@ -27,7 +27,7 @@ ext.libVersions =
     jackson_annotation: '2.5.0',
     jacoco: '0.7.7.201606060606',
     snakeyaml: '1.12',
-    cassandra: '2.2.8',
+    cassandra: '3.7',
     cassandra_driver_core: '2.2.0-rc3',
```

Comment:
If we switch one to 3.7, I think we should really be switching both. Also, cassandra 2.2.8 should support 2.2.8+ right? We may use a feature that isn't supported in 2.2.8 if we upgrade the source dependency to 3.7